ci/update-intellij/src/main/resources/intellijVersion.properties

```diff
@@ -0,0 +1,4 @@
+#Thu Aug 18 00:07:22 UTC 2022
+# Values below must match intellij version values from the reference.conf file. DO NOT CHANGE MANUALLY.
```

Comment:
Huh ideally I'd avoid repetition if possible (i.e. keep HOCON as the sole source of truth; best create a small .conf file with just the values for IntelliJ versions, that'll be easier to manage).

Reply:
You are right - unnecessary file deleted: https://github.com/VirtusLab/ide-probe/pull/280/commits/1df0b14ce3abda78a78a8f5ae7b4b519c32f759d